remote_llm.py

```diff
@@ -32,15 +33,33 @@ def generate_text(preprompt, prompt, model):
     if token is None:
         raise ValueError('LLM Verification Router token is not set')
     
-    log.info(f'Received text generation request for prompt "{prompt}" for model {model}')
+    log.info(f'Received text generation request for prompt "{prompt}" for model {model} at url {url}')
     # Load the Vanilla Neox API key from the config file.
 
     raw_response = post(url=route,
                         headers={'Authorization': f'Bearer {token}'},
-                        json={'prompt': prompt, "preprompt" : preprompt, "model": model})
+                        
+                        # UUID needs to be provided here
+                        # https://github.com/aivillage/llm_router/blob/main/src/chat/mod.rs#L35
+                        # TODO: Add assert isinstance(uuid, str) because rust is picky
+                        # json={"uuid": "1", "prompt": prompt, "preprompt" : preprompt, "model": model})
+
+                        # LLM Router fails with empty preprompt
+                        # https://github.com/aivillage/llm_router/issues/9
+                        json={"uuid": "1", "prompt": prompt, "preprompt" : "test preprompt", "model": model})
```

Comment:
Two things need to be fixed here
1. UUID needs to be provided to `generate_text`, I'm assuming it should be generated one call up so it can be logged in the DB
2. Empty preprompts are not handled by llm_router. I assume we want to allow that there